Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -8,7 +8,7 @@ namespace Neuro.Pathfinding;
 
 public sealed class PathfindingHandler
 {
-    private const float GRID_DENSITY = 6f; // TODO: Fine-tune individual maps to optimize performance
+    private const float GRID_DENSITY = 4.5f; // TODO: Fine-tune individual maps to optimize performance
     private const int GRID_BASE_WIDTH = 100;
 
```

Comment:
Same problem on polus, this area is inaccessible

However i dont think this is a problem as long as the pathfinding doesnt break when going off the node grid (because the player can spawn there). there's no reason to path above the meeting room table anyway

Resolve this comment if nothing breaks when the player spawns off the node grid.

![image](https://user-images.githubusercontent.com/32238504/230695337-0446d19d-f69e-4800-9491-3302de554902.png)

Reply:
It's a christmas miracle! Density `5f` Solved everything.

![image](https://user-images.githubusercontent.com/72096833/230702837-9ea90ab2-7d12-4124-92e4-b57c4ec8a4aa.png)